### PR TITLE
Update some function

### PR DESCRIPTION
**Pull request by Nthily**

### CODE REVIEW

File: ComposeSamples/README.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ To learn how to load apps on the Surface Duo emulator, see the [documentation](h
 
 ## Prerequisites
 
+- [Android Studio Canary](https://developer.android.com/studio/preview)
```
Comment (joyl1216) [inline ComposeSamples/README.md:21]: We don't need to state `Android Studio Canary` here, since it is mandatory for Jetpack Compose development.

Comment (Nthily) [reply ComposeSamples/README.md:21]: ok

**Comment by joyl1216** [comment]
Hi Nthily, it looks all good now. Thank you for your update. Happy to see your further contribution in the future.

**Review by joyl1216 — APPROVED** [review]

**Comment by Nthily** [comment]
thanaks !! his is also my first pr submission to a Microsoft project 🙂

**Pull request merged**